### PR TITLE
support ruby 2.1

### PR DESCRIPTION
**Pull request by andreaseger**
also adds danger

### CODE REVIEW

File: Dangerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# --------------------------------------------------------------------------------------------------------------------
+# Has any changes happened inside the actual library code?
+# --------------------------------------------------------------------------------------------------------------------
+has_app_changes = !git.modified_files.grep(/lib/).empty?
+has_test_changes = !git.modified_files.grep(/spec/).empty?
+is_version_bump = git.modified_files.sort == ["CHANGELOG.md", "lib/receptacle/version.rb"].sort
```
Comment (raphaelhetzel) [inline Dangerfile]: copy paste error, this is not receptable

File: lib/request_handler/fieldsets_parser.rb [hunk 2]
```diff
@@ -45,7 +45,7 @@ def check_required_fieldsets_types(fieldsets)
     end
 
     def raise_invalid_field_option(type)
-      return if allowed&.key?(type)
+      return if allowed && allowed.key?(type)
```
Comment (raphaelhetzel) [inline lib/request_handler/fieldsets_parser.rb]: This check if allowed exists is already checked in the base class.

File: Dangerfile (hunk 1 above)
Comment (andreaseger) [inline Dangerfile]: thanks - thought I already changed that

File: lib/request_handler/fieldsets_parser.rb (hunk 2 above)
Comment (andreaseger) [inline lib/request_handler/fieldsets_parser.rb]: true didn't see that when I was removing the lonely operators

**Pull request merged**